### ISSUE TITLE
Accessibility Audit for Keyboard Navigation, Screen Reader and High Zoom

### ISSUE THREAD

**Issue by g547315** [gchq/CyberChef#1718]
Aiming to improve the accessibility of CyberChef, an audit of keyboard navigation, screen reader use and high zoom levels has been conducted. 

The following areas for improvement have been identified.
Feedback welcome, with any feedback or suggestions on possible resolutions or additional issues, aim is to resolve these issues and any other accessibility issues identified.

### Keyboard Navigation

- Poor focus indication: Unclear navigation due to low contrast tab focus highlighting.
- Keyboard traps: Users get stuck in certain areas (e.g., input field) and cannot escape using keyboard.
- Inconsistent tabbing: Unclear and inconsistent tab flow, particularly for search, new input tab, and general navigation.
- Limited keyboard interaction: Inability to use arrow keys for basic navigation, access menus, reach tool tips, adjust input/output status, or explore top banner items.
- Missing interaction methods: Lack of keyboard alternative for drag-and-drop operations, selecting auto-bake settings, accessing output menu, and viewing output timer tool tips..
- Inaccessible documentation: Keyboard shortcut list exists but requires mouse access, this hinders discoverability.

### Screen Reader
-  Missing aria labels: Multiple components lack accessible labels, hindering screen reader users.
- Inconsistent button labels: Some buttons use confusing labels, including underscores for titles. Provide clear and concise descriptions for all buttons.

### High Zoom Levels (400%)
- Current solution fails to adequately address accessibility needs for low-vision users
